v2/cn/getting-started.md

```diff
@@ -12,12 +12,24 @@ First of all, please go to the [Download](download.html) page to get the source
 code tarball of [OpenResty](openresty.html), and see the [Installation](installation.html) page
 for how to build and install it into your system.
 
+First of all, you need to install OpenResty to your system.
+首先，你需要将 OpenResty 安装到你的系统。
+
+If you are in Linux, please check out OpenResty's official pre-built packages if your Linux distribution is currently supported. If you use the official pre-built package, please replace the nginx command in this document below with the openresty command.
+如果你使用的是 Linux 系统，请先用 OpenResty 的官方预装包，检查你的 Linux 发行版是否支持。如果你使用官方预装包，请在下面的文件里，将 Nginx 命令改为 OpenResty 命令。
```

Comment:
We need links here on the "pre-built packages". I think the original English markdown file has it?

For example, `[pre-built packages](linux-packages.html)` is a markdown link. please refer to the markdown syntax manual for more details on the markdown notation. thank you!